open-reac/src/test/resources/mock_outputs/reactiveopf_results_indic.txt

```diff
@@ -1,18 +1,18 @@
 final_status OK
 dcopf_status OK
 
-ctime_start 'Fri Nov 24 12:03:37 2023'
+ctime_start 'Mon Apr 01 12:23:50 2024'
```

Comment:
In the format exported by AMPL, this information is provided, which is questionable indeed.

For this PR, I propose to put empty dates/directory paths ("") in order to validate the TUs and make the test resource files independent. It could then be discussed to remove this information in a dedicated PR.

What do you think @So-Fras ?

Reply:
Agreed!